ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -567,6 +490,113 @@ objects:
           WORKER_SECURITY_GROUP="$(echo "${SECURITY_JSON}" | jq -r '.[] | select(.OutputKey == "WorkerSecurityGroupId").OutputValue')"
           WORKER_INSTANCE_PROFILE="$(echo "${SECURITY_JSON}" | jq -r '.[] | select(.OutputKey == "WorkerInstanceProfile").OutputValue')"
 
+          if [[ "${ENABLE_PROXY}" == "true" ]]; then
+            echo "creating proxy here..."
+            export PROXY_URL="http://admin:admin@35.196.128.173:3128/"
+          fi
+
+          # create manifests/ignition-configs
+          if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
+            MACHINE_CIDR=10.0.0.0/16
+            cat > /tmp/artifacts/installer/install-config.yaml << EOF
+          apiVersion: v1
+          baseDomain: ${BASE_DOMAIN}
+          metadata:
+            name: ${CLUSTER_NAME}
+          controlPlane:
+            name: master
+            replicas: 3
+            platform:
+              aws:
+                zones:
+                - ${AWS_REGION}a
+                - ${AWS_REGION}b
+                - ${AWS_REGION}c
+          compute:
+          - name: worker
+            replicas: 0
+            platform:
+              aws:
+                zones:
+                - ${AWS_REGION}a
+                - ${AWS_REGION}b
+                - ${AWS_REGION}c
+          networking:
+            machineCIDR: ${MACHINE_CIDR}
+          platform:
+            aws:
+              region: ${AWS_REGION}
+              userTags:
+                expirationDate: ${EXPIRATION_DATE}
+          pullSecret: >
+            ${PULL_SECRET}
+          sshKey: |
+            ${SSH_PUB_KEY}
+          EOF
+
+            # Add proxy info to install-config.yaml
+            if [[ "${ENABLE_PROXY}" == "true" ]]; then
+              cat >> /tmp/artifacts/installer/install-config.yaml << EOF
+            proxy:
+              httpsProxy: ${PROXY_URL}
+              httpProxy: ${PROXY_URL}
+            #  additionalTrustBundle: ${PROXY_CA_PATH}
+            EOF
```

Comment:
You need to ues `<<-EOF` with tabs if you want to indent here-docs.  So far we've been using `<<EOF` and leaving the heredoc at the indent level of the main file (which isn't as pretty, but whatever).